modules/git/commit.go

```diff
@@ -446,6 +446,7 @@ func (c *Commit) GetSubModule(entryname string) (*SubModule, error) {
 }
 
 // GetBranchName gets the closes branch name (as returned by 'git name-rev')
+// FIXME: This get only one branch, but one commit can be part of multiple branches
```

Comment:
Be careful. This function not only returns the first matching entry, but also doesn't check whether it's the closest to the given commit. For example:
```
dev~2
john-test1~2^1
john
john-test2~3
```
You probably want an exact match for this PR (`john`) and not any of the others.

Reply:
I'm using this function to resolve the pr. I think I'll write another function which gets me all branches a commit is part of, since it's perfectly possible one commit is part of multiple branches on multiple prs :thinking:

Reply:
I think I've come up with something appropriate here, please check again.